src/main/scala/xiangshan/cache/mmu/PageTableWalker.scala

```diff
@@ -82,6 +87,10 @@ class PTWIO()(implicit p: Parameters) extends MMUIOBaseBundle with HasPtwConst {
     val req_info = new L2TlbInnerBundle()
     val level = UInt(log2Up(Level + 1).W)
   })
+  val bitmap = new Bundle{
+    val req = DecoupledIO(new bitmapReqBundle())
+    val resp = Flipped(DecoupledIO(new bitmapRespBundle()))
+  }
```

Comment:
Should be a option bundle? Then we need not say:
```
  if (HasBitmapCheck) {
    io.bitmap.xxx := xxx
  } else {
    io.bitmap.xxx := DontCare
  }
```

If we don't set these bundles as options, we can only rely on chisel's functionality to optimise them away; not that they won't be generated initially

Reply:
The registerd are similar. They can be set to an option that is only generated when the bitmap is opened, and then they only need to be assigned values when the bitmap is detected to be open (HasBitmapExtension).